<?xml version="1.0"?>

<launch>

	<arg name="launch_zed" default="false"/>
	<arg name="resolution" default="0.1"/>

	<group if="$(arg launch_zed)">
		<include file="$(find zed_wrapper)/launch/zed.launch" />
	</group>

	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
		<param name="resolution" value="$(arg resolution)" />
		<param name="frame_id" type="string" value="/map" />
		<remap from="cloud_in" to="/zed/point_cloud/cloud_registered" />
	</node>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find drone_navigation)/rviz/octomap.rviz" output="screen" />

</launch>
